apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -311,10 +321,57 @@ func (m *ModelManagerAPI) createModelNew(
 		return modelID, errors.Annotatef(err, "failed to create model info for model %q", modelID)
 	}
 
+	// Init cloud image metadata from bootstraped data
+	if !cloud.CloudIsCAAS(m.stateParams.ControllerCloud) {
+		if err := initCloudImageMetadata(
+			ctx,
+			modelDomainServices.CloudImageMetadata(),
+			m.stateParams.ControllerModelConfig.ImageStream(),
```

Comment:
This should come from the config for the new model which may have a different value that the one set at bootstrap - even the controller model's value for image stream may be different from bootstrap.

Reply:
As discussed today:

* Behavior in 3.x is wrong, since all models share the metadata with the controller, included bootstraped metadata.
* This PR introduces separation between models, but keep the fact that some metadata can be included at bootstrap (and stored in controller model), then copied in new model (as long as any new metadata added, for some reason, to the controller model in the meanwhile)

I think it is a step forward in the right direction, but may not be the target. 

If it is actually a step in the right direction, I think we should land the PR and finalize the "right way" in another task. Main focus today is to move functionalities to dqlite, not fix long-time broken functionalities. If moving to dqlite make use closer to the right direction, it is a win.